simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/station/StationMessageSenderTest.java

```diff
@@ -51,9 +51,9 @@
     @Mock
     WebSocketClient webSocketClientMock;
 
-    StationMessageSender stationMessageSender;
+    private StationMessageSender stationMessageSender;
 
-    BlockingQueue<WebSocketClientInboxMessage> queue;
+    private BlockingQueue<WebSocketClientInboxMessage> queue;
```

Comment:
Why do you use private here? I deliberately dont use any encapsulation in the unit-tests as its just noise. Junit5 does not require encapsulation as Junit4